bindata/network/multus/multus.yaml

```diff
@@ -116,29 +117,109 @@ spec:
       tolerations:
       - operator: Exists
       serviceAccountName: multus
-      initContainers:
-      - name: multus-binary-copy
+      containers:
+      - name: kube-multus
         image: {{.MultusImage}}
-        command: ["/entrypoint/cnibincopy.sh"]
+        command: [ "/bin/bash", "-c", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh &&
```

Comment:
If we add another flag this could get confusing.  I'd recommend switching `-c` above to `-ec` and then removing this &&